TrackletAlgorithm/TrackQualityMemory.h

```diff
@@ -0,0 +1,223 @@
+#ifndef TrackletAlgorithm_TrackQualityMemory_h
+#define TrackletAlgorithm_TrackQualityMemory_h
+
+#include "ap_fixed.h"
+#include "Constants.h"
+#include "MemoryTemplate.h"
+
+// Temporary definition of TTTrack word pending changes to KF output format
+// https://twiki.cern.ch/twiki/bin/viewauth/CMS/HybridDataFormat#Final_output_to_L1_trigger
+
+// Data object definition
+class TTTrack
+{
+public:
+
+  enum BitWidths {
+    // Bit size for Track Memory fields
+    InvRSize = 15,
+    PhiSize = 12,
+    TanLSize = 16,
+    Z0Size = 12,
+    D0Size = 13,
+    Chi2rphiSize = 4,
+    Chi2rzSize = 4,
+    BendChi2Size = 3,
+    HitPatternSize = 7,
+    MVATQSize = 3,  //Track Quality
+    MVAresSize = 6, //Reserved MVA bits
+    TkValidSize = 1,
+    
+    // Bit size for full Track Memory 96-bits
+    TrackSize = InvRSize + PhiSize + TanLSize + Z0Size 
+              + D0Size + Chi2rphiSize + Chi2rzSize + BendChi2Size 
+              + HitPatternSize + MVATQSize + MVAresSize + TkValidSize + 2*kNBits_MemAddr
```

Comment:
What is the `2*kNBits_MemAddr` at the end for? By my count, this brings the width up to 110 bits.